randovania/games/dread/exporter/patch_data_factory.py

```diff
@@ -486,6 +498,16 @@ def _tilegroup_patches(self):
             }
         ]
 
+    def _light_patches(self):
+        config = self.configuration.disabled_lights.as_json
+        patches = []
+
+        for region, is_disabled in config.items():
+            if is_disabled:
+                patches.append({"scenario": _SCENARIO_IDS[region], "actor_layer": "rLightsLayer", "method": "all"})
```

Comment:
```suggestion
        for region_name, is_disabled in config.items():
            if is_disabled:
                scenario_id = self.game.region_list.region_with_name(region_name.capitalize()).extra.get("scenario_id")
                patches.append({"scenario": scenario_id, "actor_layer": "rLightsLayer", "method": "all"})
```

Reply:
Thanks, I was looking for something like this method but couldn't find it.